### ISSUE TITLE
Feature request: Support for es6 modules

### ISSUE THREAD

**Issue by thematho** [chaijs/chai-as-promised#280]
Hi,
Currently there is no support for ES6 Imports, I found [this closed issue #133](https://github.com/domenic/chai-as-promised/issues/133), mentionening that es6 module support will be implemented at one point.

Example:
`import * as chaiAsPromised from 'chai-as-promised';` or
`import { chaiAsPromised } from 'chai-as-promised';`

Actual error:
```
SyntaxError: The requested module './index.js' does not provide an export named 'default'
 ❌ Could not import your test module. Check the browser logs or open the browser in debug mode for more information. 
```

**Comment by mcm1957** [comment]
As far as I know js-controller currently does not support ESM at all. 
Work for ESM exports has started - bit will likely not be available before js-controller 6
A solution fpr compact moide is still not found.
@foxrive76 please orrect if I'm wrong

So this is - as far as I know - currently NO issue for adapter creator but for the iobroker core code.

**Comment by 43081j** [comment]
Closed by #287 (though yet to be released in 8.0.0)
